lib/kafka_ex/network/tcp.ex

```diff
@@ -0,0 +1,19 @@
+defmodule KafkaEx.Network.TCP do
```

Comment:
Dependency injection, for testing purposes. If you look at https://github.com/wooga/kafka_ex/blob/31cd90697729fbe1f6eba8ef86a2eff512dd6232/test/network_client_test.exs you can see it being replace with a mock module that doesn't interact with real sockets at all.